core/unix/os.c

```diff
@@ -8482,11 +8497,19 @@ get_dynamo_library_bounds(void)
     char *libdir;
     const char *dynamorio_libname;
 #ifdef STATIC_LIBRARY
-    /* We don't know our image name, so look up our bounds with an internal
-     * address.
-     */
+    bool have_linker_vars = &dynamorio_so_start != &weak_dynamorio_so_bounds_filler &&
```

Comment:
Please move the declaration and comment about address-only (lines 8515-8519 in your CL; Travis won't let me put a comment there!) up above the ref here.  Prob want to tweak the comment.

Reply:
Good catch, I meant to do this. I put the external declarations for all of LINUX at the top level and move the warning to those lines.